backend/grant-service/api/src/main/kotlin/Grants.kt

```diff
@@ -822,6 +854,26 @@ object Grants : CallDescriptionContainer("grant") {
         }
     }
 
+    val findAffiliations = call<FindAffiliationsRequest, FindAffiliationsResponse, CommonErrorMessage>("FindAffiliations") {
+        auth {
+            access = AccessRight.READ
+        }
+        http {
+            method = HttpMethod.Get
+
+            path {
+                using(baseContext)
+                +"find-affiliations"
+            }
+
+            params {
+                +boundTo(FindAffiliationsRequest::username)
```

Comment:
Should be replaced with a grant ID such that we can enforce a proper security authorization check. Only a grant approver who has received an application should be able to lookup this information regarding a different user.